### DISCUSSION TITLE
Windows and --live=code

### DISCUSSION THREAD

**Discussion by jeffmaury** [jbangdev/jbang#322] [General]
I have a jbang file and running on Windows10 Pro in the Windows shell.
If I run `code zenhub.java` the code is launched
If I run `jbang edit --live=code zenhub.java` then I have:

```
[jbang] Running `code C:\Users\JeffMAURY\.jbang\cache\projects\zenhub.java_jbang_c658d0811def722f8523f7570592602670a654ed66e81ef74186a9ee1a0edea9\zenhub`
[jbang] java.io.IOException: Cannot run program "code": CreateProcess error=2, Le fichier spécifié est introuvable
[jbang] Run with --verbose for more details
```

Am I missing something or do we have a Windows issue

**Comment by maxandersen** [comment]
on windows the command to launch with is `jbang edit --live=code.cmd zenhub.java` i believe.

I probably should change the launch code to use shell on linux and `cmd.exe /C` on windows so windows own implicit logic for executables apply and on linux aliases should work too.

But for now you need to be explicit.

**Comment by jeffmaury** [comment]
Works with `--live=code.cmd` now.
However, when I exit from code, jbang i still running and I don't get back to the Windows shell

**Reply by maxandersen** [reply]
yes, code.cmd (and other tools) forks separate process for the editor/ide thus we can't know when is the right time to exit. ctrl+c would be neeed.

**Comment by maxandersen** [comment]
fyi, in release v0.48 we now launch using the shell so `jbang edit --open=code xyz.java` works the same on all platforms.
